CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 
 ### Fixed
 
+- Fixed a bug that prevented the `--show-log-origin` option from working.
```

Comment:
what what the real impact of the bug for a customer? can we expand on what was not working? 
example 1: (i know it was not the problem here but the concept is the same)  "Fixed a bug that ignored <option> to be parsed , consequentially not printing the source code file of the log message."
example 2: "Fixed a bug that prevented to display the source file (or poisoned the source file with Y) from which a log is generated when using `--show-log-origin` option" or something like that.

Reply:
Ok, I was assuming familiarity with the option. I will amend.

Reply:
Fixed, let me know if that is better.